sabnzbd/misc.py

```diff
@@ -322,11 +323,29 @@ def set_serv_parms(service, args):
     return True
 
 
+def set_socks5_proxy():
+    if cfg.socks5_proxy_url():
+        proxy = urllib.parse.urlparse(cfg.socks5_proxy_url())
+        logging.debug("Using proxy %s:%s", proxy.hostname, proxy.port)
```

Comment:
I didn't think of using `urllib`, smart!
Doesn't the library also support other type of proxies, or is only socks5 a relevant one?

Reply:
SOCKS4 came four years before SOCKS5, which came 25 years ago. I really doubt anything supports 4 and not 5 these days.